app/src/main/java/com/duycuong/weather/ui/screen/main/MainViewModel.java

```diff
@@ -1,6 +1,7 @@
 package com.duycuong.weather.ui.screen.main;
 
 import android.content.Context;
+import android.content.SharedPreferences;
```

Comment:
ERROR: Unused import: android.content.SharedPreferences. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports